src/importexport/mei/internal/meiconverter.cpp

```diff
@@ -866,6 +866,8 @@ libmei::Clef Convert::clefToMEI(engraving::ClefType clef)
     case (engraving::ClefType::G8_VB):
     case (engraving::ClefType::G8_VA):
     case (engraving::ClefType::G15_MA):
+    case (engraving::ClefType::G8_VB_C):
+    case (engraving::ClefType::G8_VB_P):
```

Comment:
@rettinghaus, @lpugin please double check

Reply:
No need to add them there. Already handled as default.

Reply:
OK, removed